portfolio/src/main/java/com/google/sps/servlets/ServletSize.java

```diff
@@ -0,0 +1,56 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import com.google.sps.usercomment.UserComments;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.FetchOptions;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.*;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+@WebServlet("/comments-size")
+public class ServletSize extends HttpServlet {
+
+  private int numberOfComments;
+
+  public int loadComments() throws IOException {
+    Query commentsQuery = new Query("Comment"); // Get previous stored comments
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    
+    // Prepare to instance the past stored comments
+    PreparedQuery comments = datastore.prepare(commentsQuery);
+
+    return comments.countEntities();
+  }
+  
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("text/html;");
+    response.getWriter().println(loadComments());
+  }
+}
```

Comment:
I think you can handle this whole servlet in the same servlet where you load the comments. If you load the comments in one servlet, you can easy count the numbers of comments there and eliminate this extra request.

Reply:
I think it they should stay in different servlets. The idea of this servlet is to get the maximum length of comments the user can ask for, but not actually loading them. In that case I will have to load the comments and then make a second call the LoadComments servlet to get the trimmed array the user asked for.